server/events/stale_command_handler.go

```diff
@@ -0,0 +1,25 @@
+package events
+
+import (
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/uber-go/tally"
+)
+
+type StaleCommandHandler struct {
+	Counter tally.Counter
```

Comment:
I would just pass the stats object here because when i look at this its not clear to me what Counter actually does unless i read through the implementation. Maybe that's a naming thing? Even naming this like `StaleCommandCounter` would be better i think.

Reply:
Replaced `Counter` with a`StaleStatsScope` and will generated counter within function instead.